T: 0.05
N: 99

parallel_park:
  offset_x = 0.63
  offset_y = 0.32
  target_yaw = 3.14159265359

constraints:
  v_max: 0.7
  v_min: -0.7
  v_ref: -0.7
  delta_max: 0.4
  delta_min: -0.4
  x_min: -2
  x_max: 2
  y_min: -2
  y_max: 2

costs:
  x_cost: 1
  y_cost: 7
  yaw_cost: 1
  v_cost: 0.5
  steer_cost: 0.05
  delta_v_cost: 0.
  delta_steer_cost: 0.